ops/dev/container_instances.tf

```diff
@@ -7,7 +7,6 @@ module "db_rollback" {
   env                     = local.env
   resource_group_name     = data.azurerm_resource_group.rg.name
   resource_group_location = data.azurerm_resource_group.rg.location
-  network_profile_id      = data.terraform_remote_state.persistent_dev.outputs.network_profile_id
```

Comment:
Can I ask why this is being removed?

Reply:
This field this variable is used for has been deprecated by azure. We are able to pass in subnet ids if required, but when I tested the rollback job in dev2 I didn't find any issues with the job when leaving the subnet ids out.

I do plan on testing the rollback job one more time before rollout and I do intend to keep this in mind for that test. Do you have any preferences around this?